.github/ISSUE_TEMPLATE/api_references_issue.md

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
I wouldn't add an API reference issue template. 
If we were to add one at some point we'd need to add it to the main repo, not the website repo, since the API reference code is in the main repo